pom.xml

```diff
@@ -109,9 +109,6 @@ limitations under the License.
             <url>https://github.com/twilmes</url>
         </contributor>
     </contributors>
-    <prerequisites>
-        <maven>3.3.9</maven>
```

Comment:
any reason you removed 3.3.9? 

it's interesting that we have 3.3.9 at the maven level, but enforcer only requires 3.2.5 (and our dev docs also say the minimum is 3.2.5).  I noticed that the reason we went to a 3.3.9 minimum was something related to the jacoco plugin requiring that version:

https://github.com/apache/tinkerpop/commit/5c9c8ce1e602c7766f4f7200ba5ec4e66718fbc5

I wonder if we should really be at 3.3.9 rather than 3.2.5...thoughts?